x-pack/plugins/osquery/server/routes/live_query/get_live_query_results_route.ts

```diff
@@ -88,10 +90,12 @@ export const getLiveQueryResultsRoute = (router: IRouter<DataRequestHandlerConte
                   request.query.page ?? 0,
                   request.query.pageSize ?? 100
                 ),
-                sort: {
-                  direction: request.query.sortOrder ?? 'desc',
-                  field: request.query.sort ?? '@timestamp',
-                },
+                sort: [
+                  {
+                    direction: (request.query.sortOrder as Direction) ?? Direction.desc,
```

Comment:
This doesn't really fix anything except replace hardcoded `desc` with the const we already have. Right?

Reply:
It changes the way Sort is provided: 
Before it was an Object {direction, field} now it's an array {direction, field}[]

Reply:
Indeed. I missed that.